apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -32,7 +32,7 @@ public class TestDatabaseFixture : IDisposable, ICollectionFixture<TestDatabaseF
     {
         private readonly Lazy<Task> _creationTask;
         public string TestInstanceName => "spannerintegration";
-        public string TestProjectName => Environment.GetEnvironmentVariable("TEST_PROJECT") ?? "cloud-sharp-jenkins";
+        public string TestProjectName => Environment.GetEnvironmentVariable("TEST_PROJECT");
```

Comment:
I'd actually assign this in the constructor, failing if it's not set.

Maybe it's time to add it to ClientTesting? It's everywhere at the moment, and ClientTesting is now a dependency in all test packages. I'm happy to do that if you'd like.

Reply:
Yea I actually have a util for this in all of my tests (which I think I copied from one of the other projects).

I can close this PR and we can make this a common method in ClientTesting.  

@jskeet I'm happy to give this a go but won't complain if you want to do it. :)  Just let me know.  I can probably do it today.

https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Utils.cs#L30

Reply:
I'll do it on Monday unless you have any kind of urgency - that's fine.

Reply:
Cool, SGTM, closing this PR.